<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThinSectionAnalysisDialog</class>
 <widget class="QDialog" name="ThinSectionAnalysisDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblDirectory">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOpenDir">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QListWidget" name="lstFiles">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragOnly</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>plane polarized images</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>cross polarized images</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QListWidget" name="lstPlanePolarizationImages">
           <property name="dragDropMode">
            <enum>QAbstractItemView::DropOnly</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QListWidget" name="lstCrossPolarizationImages">
           <property name="dragDropMode">
            <enum>QAbstractItemView::DropOnly</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblImgPlanePolarization">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="lblImgCrossPolarization">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>HSI mode:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmbHSImode">
       <item>
        <property name="text">
         <string>HSV</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HSL</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>δ:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="dblSpinSimilarityThreshold">
       <property name="toolTip">
        <string>Similarity threshold (δ).</string>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.850000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRun">
       <property name="toolTip">
        <string>Execute analysis.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons32/play32</normaloff>:/icons32/play32</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDismiss">
       <property name="text">
        <string>Dismiss</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnDismiss</sender>
   <signal>clicked()</signal>
   <receiver>ThinSectionAnalysisDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOpenDir</sender>
   <signal>clicked()</signal>
   <receiver>ThinSectionAnalysisDialog</receiver>
   <slot>onOpenDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstPlanePolarizationImages</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>ThinSectionAnalysisDialog</receiver>
   <slot>onPlanePolarizationImageSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lstCrossPolarizationImages</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>ThinSectionAnalysisDialog</receiver>
   <slot>onCrossPolarizationImageSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRun</sender>
   <signal>clicked()</signal>
   <receiver>ThinSectionAnalysisDialog</receiver>
   <slot>onRun()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOpenDir()</slot>
  <slot>onPlanePolarizationImageSelected()</slot>
  <slot>onCrossPolarizationImageSelected()</slot>
  <slot>onRun()</slot>
 </slots>
</ui>
